rav1e_wasm/README.md

```diff
@@ -0,0 +1,34 @@
+# rav1e_wasm
+
+- [🚴 Usage](#-usage)
+  - [Example Website](#example-website)
+- [🔋 Batteries Included](#-batteries-included)
+
+## 🚴 Usage
+
+```bash
+# 🛠️ Build with `wasm-pack build`
+wasm-pack build
+
+# 🔬 Test in Headless Browsers with `wasm-pack test`
+wasm-pack test --headless --firefox
+```
+
+### Example Website
+```bash
+rav1e_wasm$ wasm-pack build
+rav1e_wasm$ cd www-example/
+rav1e_wasm/www$ npm install
+rav1e_wasm/www$ npm run start
+# website served at localhost:8080
```

Comment:
Either keep it or remove it, having commented one seems odd.

Reply:
> Also, we would require the CI integration for the npm thing for making this patchset complete

I am a bit unsure about how to tag the package. Should I just always increment the version? Or is it possible to use the commit-hash as version?

Best handling imo would be to keep the rav1e-rs and rav1e-npm version in sync, but I don't really see how this is possible.